bootstrap/bootstrap.py

```diff
@@ -0,0 +1,119 @@
+                                       dir, filename.replace(strtoreplace, self._project_name, 1))  # NOQA: E501
+                    os.rename(src, dst)
+
+    def renamedir(self):
+        # Rename any directory with diabetes_regression with project name
+        dirs = ["diabetes_regression"]
+        for dir in dirs:
+            src = os.path.join(self._project_directory, dir)
+            dst = os.path.join(self._project_directory, self._project_name)
+            os.rename(src, dst)
+
+    def deletedir(self):
+        # Delete unwanted directories
+        dirs = [".git", "docs", r"diabetes_regression\training\R"]
+        for dir in dirs:
+            os.system(
+                'rmdir /S /Q "{}"'.format(os.path.join(self._project_directory, dir)))  # NOQA: E501
+
+    def replaceimport(self):
+        # Replace imports with new project name
+        dirs = [r"diabetes_regression\training\test_train.py",
+                r"ml_service\pipelines\diabetes_regression_verify_train_pipeline.py"]  # NOQA: E501
+        for file in dirs:
+            fin = open(os.path.join(self._project_directory, file), "rt")
+            data = fin.read()
+            newimport = "from " + self._project_name + "."
+            data = data.replace("from diabetes_regression.", newimport)
+            fin.close()
+            fin = open(os.path.join(self._project_directory, file), "wt")
+            fin.write(data)
+            fin.close()
+
+    def cleandir(self):
+        # Clean up directories
+        dirs = ["data", "experimentation"]
+        for dir in dirs:
+            for root, dirs, files in os.walk(os.path.join(self._project_directory, dir)):  # NOQA: E501
+                for file in files:
+                    os.remove(os.path.join(root, file))
+
+    def validateargs(self):
+        # Validate arguments
+        if (os.path.isdir(self._project_directory) is False):
+            raise Exception(
+                "Not a valid directory. Please provide absolute directory path")  # NOQA: E501
+        if (len(os.listdir(self._project_directory)) > 0):
+            raise Exception("Directory not empty. PLease empty directory")
+        if(len(self._project_name) < 3 or len(self._project_name) > 8):
+            raise Exception("Project name should be 3 to 8 chars long")
+
+
+def main(args):
+    # Run this script to create a template from mlopspython
+    #  python bootstrap.py --d [dirpath] --n [projectname]
+    parser = argparse.ArgumentParser(description='New Template')
+    parser.add_argument("--d", type=str,
+                        help="Absolute path to new project direcory")
+    parser.add_argument(
+        "--n", type=str, help="Name of the project[3-8 chars] ")
+    try:
+        args = parser.parse_args()
+        project_directory = args.d
+        project_name = args.n
+        helper = Helper(project_directory, project_name)
+        helper.validateargs()
+        helper.clonerepo()
```

Comment:
Could we add an argument like `skipClone` which would do everything you already do minus clone to a new dir?  It would then allow people to use the repo as a code template as well and just run the bootstrap script after the fact.  I think you'd only put a conditional around this and deleting the `.git` directory.